src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard-area-chart/dashboard-area-chart.component.ts

```diff
@@ -300,6 +355,6 @@ export class DashboardAreaChartComponent implements OnChanges {
       return `${value} ${maxValueDataUnits}`;
     };
     this.chartDataUnits = maxValueDataUnits || '';
-    this.chart.chart.update();
+    this.cdr.detectChanges();
```

Comment:
curious, why did we removed the native chart update function and went for a force change detection?

Reply:
Updated..it was just for some testing